packages/desktop-client/src/components/common/Search.tsx

```diff
@@ -65,19 +79,7 @@ export default function Search({
           </Button>
         )
       }
-      inputStyle={{
-        '::placeholder': {
-          color: theme.formInputTextPlaceholder,
-          transition: 'color .25s',
-        },
-        ':focus': isInModal
-          ? null
-          : {
-              '::placeholder': {
-                color: theme.formInputTextPlaceholderSelected,
-              },
-            },
-      }}
+      inputStyle={inputStyle}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is this change necessary?